pkg/skaffold/docker/context_test.go

```diff
@@ -19,51 +19,59 @@ package docker
 import (
 	"archive/tar"
 	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"testing"
 
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
-	"github.com/pkg/errors"
+	"github.com/GoogleContainerTools/skaffold/testutil"
 )
 
 func TestDockerContext(t *testing.T) {
+	tmpDir, cleanup := testutil.TempDir(t)
+	defer cleanup()
+
+	os.Mkdir(filepath.Join(tmpDir, "files"), 0750)
+	ioutil.WriteFile(filepath.Join(tmpDir, "files", "ignored.txt"), []byte(""), 0644)
+	ioutil.WriteFile(filepath.Join(tmpDir, "files", "included.txt"), []byte(""), 0644)
+	ioutil.WriteFile(filepath.Join(tmpDir, ".dockerignore"), []byte("**/ignored.txt"), 0644)
```

Comment:
Can we add a test case for #386. 
Add a line to .dockerignore with "alsoignored.txt" and create a file alsoignored.txt.

Reply:
I'll do that, thanks for the idea!